benchmarks/benchmarks.py

```diff
@@ -51,16 +51,16 @@ def setup(self):
         # Stuff that depends on the recomb_map_chr22 will fail
         if stdpopsim_available:
             species = stdpopsim.get_species("HomSap")
-            genetic_map = species.get_genetic_map("HapMapII_GRCh37")
```

Comment:
Why are you changing this? I think what you're doing will result in a uniform map, which takes away the main point